server/utils/premises/occupancy.ts

```diff
@@ -88,10 +88,10 @@ export const generateDaySummaryText = (daySummary: Cas1PremisesDaySummary): stri
     )
     .filter(Boolean)
   const messages: Array<string> = []
-  if (bookingCount > availableBedCount) messages.push('has bookings exceeding its available capacity')
+  if (bookingCount > availableBedCount) messages.push('is overbooked')
   if (overbookedCriteria.length)
     messages.push(
-      `is overbooked on spaces with the following ${overbookedCriteria.length > 1 ? 'criteria' : 'criterion'}: ${overbookedCriteria.map(characteristic => occupancyCriteriaMap[characteristic].toLowerCase()).join(', ')}`,
+      `is overbooked on: ${joinWithCommas(overbookedCriteria.map(characteristic => occupancyCriteriaMap[characteristic].toLowerCase()))}`,
     )
   return messages.length ? `This AP ${messages.join(' and ')}.` : ''
```

Comment:
If the premises is overbooked on basic rooms _and also_ on specific criteria, the message would end up reading `This AP is overbooked and is overbooked on: en-suite [...]` -- doesn't sound right?!